packages/elements/src/components/logto-button.styles.ts

```diff
@@ -0,0 +1,106 @@
+import { css } from 'lit';
+
+import { unit } from '../utils/css.js';
+import { vars } from '../utils/theme.js';
+
+export const buttonSizes = css`
+  :host([size='small']) {
+    height: 30px;
+    padding: ${unit(0, 3)};
+  }
+
+  :host([size='small'][type='text']) {
+    height: 24px;
+  }
+
+  :host([size='medium']) {
+    height: 36px;
+    padding: ${unit(0, 4)};
+  }
+
+  :host([size='medium'][type='text']) {
+    height: 28px;
+    font: ${vars.fontLabel1};
+  }
+
+  :host([size='large']) {
+    height: 44px;
+    padding: ${unit(0, 6)};
+  }
+
+  :host([size='large'][type='text']) {
+    height: 28px;
+    font: ${vars.fontLabel1};
+  }
+`;
+
+export const textButton = css`
+  :host([type='text']) {
```

Comment:
Will the `:host` decorator impact the styles globally?

Reply:
you can double check MDN, it's kind of Web Components 101

Reply:
OK, will take a look